express/scripts/scripts.js

```diff
@@ -2138,6 +2143,15 @@ export async function addFreePlanWidget(elem) {
       <div><div>${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
       <div><div>${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
     `;
+    document.addEventListener('planscomparisonloaded', () => {
```

Comment:
Why is that block specific code in scripts.js ?

Reply:
This is for the free-plan widget to load the nav button to the block on pages that have loaded the block. I'd be happy to look into implementing it within the block if this isn't an appropriate place for it.

Reply:
I have just realised this is in a `addFreePlanWidget` which is already block specific. This would require more changes. Let's merge it as is for now.